src/sqlfluff/core/linter/runner.py

```diff
@@ -37,7 +37,7 @@ def __init__(
 
     pass_formatter = True
 
-    def iter_rendered(self, fnames: List[str]) -> Iterator[Tuple]:
+    def iter_rendered(self, fnames: List[str]) -> Iterator[Tuple[str, RenderedFile]]:
```

Comment:
What's this change? Was the existing type annotation wrong?

Reply:
It wasn't wrong, just a bit vague. A more specific one would have been useful in debugging so I thought I'd improve it while I was there.